sdcm/cluster_aws.py

```diff
@@ -1374,7 +1374,7 @@ def __init__(
         self.scylla_cluster = None
 
         node_prefix = cluster.prepend_user_prefix(user_prefix, "vs-node")
-        node_type = "vs"
+        node_type = "vector-store"
```

Comment:
smthing unrelated got in after last rebase - changes around vs and logging

Reply:
my bad, I've amend the last commit during rebase..

fixing it in a sec.